apps/server/src/api-data/excel/excel.parser.ts

```diff
@@ -63,166 +68,75 @@ export const parseExcel = (
-      },
-      [importMap.timeEnd]: (row: number, col: number) => {
-        timeEndIndex = col;
-        rundownMetadata['timeEnd'] = { row, col };
-      },
-      [importMap.duration]: (row: number, col: number) => {
-        durationIndex = col;
-        rundownMetadata['duration'] = { row, col };
-      },
-
-      [importMap.cue]: (row: number, col: number) => {
-        cueIndex = col;
-        rundownMetadata['cue'] = { row, col };
-      },
-      [importMap.title]: (row: number, col: number) => {
-        titleIndex = col;
-        rundownMetadata['title'] = { row, col };
-      },
-      [importMap.flag]: (row: number, col: number) => {
-        flagIndex = col;
-        rundownMetadata['flag'] = { row, col };
-      },
-      [importMap.countToEnd]: (row: number, col: number) => {
-        countToEndIndex = col;
-        rundownMetadata['countToEnd'] = { row, col };
-      },
-      [importMap.skip]: (row: number, col: number) => {
-        skipIndex = col;
-        rundownMetadata['skip'] = { row, col };
-      },
-      [importMap.note]: (row: number, col: number) => {
-        notesIndex = col;
-        rundownMetadata['note'] = { row, col };
-      },
-      [importMap.colour]: (row: number, col: number) => {
-        colourIndex = col;
-        rundownMetadata['colour'] = { row, col };
-      },
-      [importMap.endAction]: (row: number, col: number) => {
-        endActionIndex = col;
-        rundownMetadata['endAction'] = { row, col };
-      },
-      [importMap.timerType]: (row: number, col: number) => {
-        timerTypeIndex = col;
-        rundownMetadata['timerType'] = { row, col };
-      },
-      [importMap.timeWarning]: (row: number, col: number) => {
-        timeWarningIndex = col;
-        rundownMetadata['timeWarning'] = { row, col };
-      },
-      [importMap.timeDanger]: (row: number, col: number) => {
-        timeDangerIndex = col;
-        rundownMetadata['timeDanger'] = { row, col };
-      },
-      [importMap.entryId]: (row: number, col: number) => {
-        entryIdIndex = col;
-        rundownMetadata['id'] = { row, col };
-      },
-      custom: (row: number, col: number, columnText: string, ontimeKey: string) => {
-        customFieldIndexes[col] = columnText;
-        rundownMetadata[`custom:${ontimeKey}`] = { row, col };
-      },
-    } as const;
+    const entry: Partial<MergedOntimeEntry> = {};
 
-    const entry: Partial<Merge<OntimeEvent, OntimeGroup>> = {};
     const entryCustomFields: EntryCustomFields = {};
 
     for (let j = 0; j < row.length; j++) {
       const column = row[j];
       // 1. we check if we have set a flag for a known field
-      if (j === timerTypeIndex) {
-        const maybeTimeType = makeString(column, '');
-        if (maybeTimeType === 'group') {
-          // we leave this as a clue for the object filtering later on
+      if (j === indexMap.timerType) {
+        const maybeTimeType = makeString(column, '').toLocaleLowerCase();
```

Comment:
I believe it is better to use the normal `lowerCase` so we have predictable results. `localeLowerCase` can be OK for user facing text

```suggestion
        const maybeTimeType = makeString(column, '').toLowerCase();
```